app/src/main/java/com/celdevLabs/foodie_app/db/entities/Dish.kt

```diff
@@ -15,7 +15,8 @@ data class Dish (
     var date: Long,
     var location: String,
     var notes: String,
-    val dishUri: String
+    val dishUri: String,
+    val rating: Double
```

Comment:
Does this need to be a `Double`? Can it just be a float? In the grand scheme of things it doesn't matter since a 32-bit float can hold the rating value just fine but just want you to consider that. Especially, if the view you are using in the end is expecting a float, might as well use a float all the way through.